src/components/Requirements.vue

```diff
@@ -1,149 +1,199 @@
 <template v-if="semesters">
-  <div class="requirements">
-    <div class="fixed">
-    <h1 class="title">School Requirements</h1>
+  <div class= "section">
+  <div v-if="isEditing" class= "editing">
```

Comment:
Could you leave a comment at the top of this section just saying that this is the editing section if we were to implement it later, but right now this section never displays?